src/services/__tests__/helpers.spec.ts

```diff
@@ -51,9 +51,8 @@ describe('helpers', () => {
     it('should open first exercise of second discipline, if first discipline was finished yet', async () => {
       mocked(loadDiscipline).mockReturnValueOnce(Promise.resolve(mockDisciplines(true)[0]))
```

Comment:
Some of these tests here, that you adjusted, do not what they should do. Read the title carefully, what the test should check, and the check if the testdata matches the expected output.